src/visualization.jl

```diff
@@ -7,10 +7,10 @@ mutable struct CWorldVis
 end
 
 function CWorldVis(w::CWorld;
-                   s=nothing,
-                   f=nothing,
-                   g=nothing,
-                   title=nothing)
+                   s::Union{Vec2, Nothing}=nothing,
+                   f::Union{Function, Nothing}=nothing,
+                   g::Union{AbstractGrid, Nothing}=nothing,
+                   title::Union{String, Nothing}=nothing)
```

Comment:
Type annotations for function arguments should usually be left off unless they are needed for dispatch. In the new code, `s=[1,2]` will cause an error, whereas previously it would automatically convert it.